app/controllers/declaration/BorderTransportController.scala

```diff
@@ -38,9 +39,12 @@ class BorderTransportController @Inject()(
   journeyType: JourneyAction,
   errorHandler: ErrorHandler,
   customsCacheService: CustomsCacheService,
+  exportsCacheService: ExportsCacheService,
   mcc: MessagesControllerComponents
 )(implicit ec: ExecutionContext, appConfig: AppConfig)
-    extends FrontendController(mcc) with I18nSupport {
+    extends {
+  val cacheService = exportsCacheService
```

Comment:
Why have we done this in the extends block?  you can just have `override val cacheService: ExportsCacheService` as a constructor arg

Reply:
This was just a style that we decided to go with on our Mob session.

Reply:
Fair enough, it's a little more verbose but does the same thing I guess.
I think a side effect will be that in your `BorderTransportController` you can access the `ExportsCacheService` using the variable name `cacheService` or `exportsCacheService`  which might be a little confusing